packages/bot-web-ui/src/components/dashboard/tutorial-tab/guide-content.tsx

```diff
@@ -27,10 +28,24 @@ const GuideContent = ({
     setTourDialogVisibility,
 }: TGuideContent) => {
     const triggerTour = (type: string) => {
+        const storage = JSON.parse(localStorage?.dbot_settings);
         if (type === 'OnBoard') {
-            setActiveTab(0);
+            if (storage.onboard_tour_token !== undefined) {
```

Comment:
@farabi-deriv can be  `storage.onboard_tour_token` equal  `null`? if not then you can do it shorter:
`if (storage.onboard_tour_token) {...`

Reply:
Right. Will make it shorter

Reply:
Done.